static/app/components/replays/replayController.tsx

```diff
@@ -0,0 +1,196 @@
+
+import Button from 'sentry/components/button';
+import ButtonBar from 'sentry/components/buttonBar';
+import BooleanField from 'sentry/components/forms/booleanField';
+import RangeSlider from 'sentry/components/forms/controls/rangeSlider';
+import {Consumer as ReplayContextProvider} from 'sentry/components/replays/replayContext';
+import {IconPause, IconPlay, IconRefresh, IconResize} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+
+import FlexLayout from './flexLayout';
+import useRAF from './useRAF';
+import {formatTime} from './utils';
+
+const SECOND = 1000;
+
+function useCurrentTime(callback: () => number) {
+  const [currentTime, setCurrentTime] = useState(0);
+  useRAF(() => setCurrentTime(callback));
+  return currentTime;
+}
+
+interface ReplayControllerProps {
+  onFullscreen?: () => void;
+  speedOptions?: number[];
+}
+
+interface ControlsProps extends Required<ReplayControllerProps> {
+  duration: number | undefined;
+  getCurrentTime: () => number;
+  isPlaying: boolean;
+  isSkippingInactive: boolean;
+  onChangeSpeed: (value: number) => void;
+  setCurrentTime: (time: number) => void;
+  speed: number;
+  togglePlayPause: (play: boolean) => void;
+  toggleSkipInactive: (skip: boolean) => void;
+}
+
+const ReplayControls = ({
+  duration,
+  getCurrentTime,
+  isPlaying,
+  isSkippingInactive,
+  onChangeSpeed,
+  onFullscreen,
+  setCurrentTime,
+  speed,
+  speedOptions,
+  togglePlayPause,
+  toggleSkipInactive,
+}: ControlsProps) => {
+  const currentTime = useCurrentTime(getCurrentTime);
+
+  return (
+    <FlexLayout direction="column" gap={0}>
+      <TimelineRange
+        data-test-id="replay-timeline-range"
+        name="repaly-timeline"
+        min={0}
+        max={duration}
+        value={Math.round(currentTime)}
+        onChange={value => setCurrentTime(value || 0)}
+        showLabel={false}
+      />
+      <ButtonLayout>
+        <FlexLayout direction="row" alignItems="center">
+          <ButtonBar merged>
+            <Button
+              data-test-id="replay-back-10s"
+              size="xsmall"
+              title={t('Go back 10 seconds')}
+              icon={<IconRefresh color="gray500" size="sm" />}
+              onClick={() => setCurrentTime(currentTime - 10 * SECOND)}
+              aria-label={t('Jump back 10 seconds')}
+            />
+            <Button
+              data-test-id="replay-play-pause"
+              size="xsmall"
+              title={isPlaying ? t('Pause the Replay') : t('Play the Replay')}
+              icon={isPlaying ? <IconPause size="sm" /> : <IconPlay size="sm" />}
+              onClick={() => togglePlayPause(!isPlaying)}
+              aria-label={t('Toggle real-time updates')}
+            />
+            <Button
+              data-test-id="replay-forward-10s"
+              size="xsmall"
+              title={t('Go forwards 10 seconds')}
+              icon={<IconRefresh color="gray500" size="sm" />}
```

Comment:
This is the wrong icon, but i couldn't find a clock-wise arrow in the storybook examples

This is what it looks like now:
<img width="309" alt="Screen Shot 2022-04-08 at 12 02 37 PM" src="https://user-images.githubusercontent.com/187460/162507863-df4ccb86-c262-4cea-8614-a8ed9b5751f4.png">